core/designsystem/src/main/kotlin/com/naveenapps/expensemanager/core/designsystem/ui/extensions/ContextExt.kt

```diff
@@ -29,6 +30,9 @@ fun openWebPage(context: Context, webpage: String) {
         context.startActivity(intent)
     } catch (e: ActivityNotFoundException) {
         // Define what your app should do if no activity can handle the intent.
+        Toast.makeText(context, context.getString(R.string.no_browser_found), Toast.LENGTH_SHORT)
```

Comment:
Hey @Tonnie-Dev, This method does not handle the browser. It seems and Am I missing anything here?